_posts/2019-11-01-understanding-dart-sass-modules-and-name-spaced-variables.md

```diff
@@ -0,0 +1,185 @@
+---
+layout: post
+title: "Understanding Dart Sass modules and namespaced variables"
+date: 2019-11-01 11:48:26
+tags:
+- Sass
+- Dart Sass
+categories: CSS
+twitter_text: "Understanding Dart Sass modules and name spaced variables"
+authors: Kevin Hougasian
+image: https://images.unsplash.com/photo-1496715976403-7e36dc43f17b?w=1000
+image_url: https://unsplash.com/photos/W8KTS-mhFUE
+image_credit: "@billy_huy"
+---
+
+For a long time, we were dependent on the CSS frameworks of others -- and we were quite happy. 😀
+
+As our needs grew, we needed a framework that suited all of them. So we created [Platform UI](https://style.rimdev.io), our custom framework written Sass, BEM, and vanilla JS for our static sites and application platform.
+
+Sass recently launched the new [Dart Sass module system](https://github.com/sass/dart-sass/releases/tag/1.23.0), which [Miriam Suzanne](@mirisuzanne) did a great job of introducing in a recent [CSS-Tricks post](https://css-tricks.com/introducing-sass-modules/).
```

Comment:
Do we have something to make `@mirisuzanne` a link? Or, can you change it to a Twitter link?

Reply:
jekyll wil l convert the twitter handle on it's own.
thanks for the other edits! :D